app/src/main/java/com/jnj/vaccinetracker/register/screens/RegisterParticipantParticipantDetailsViewModel.kt

```diff
@@ -65,6 +65,21 @@ class RegisterParticipantParticipantDetailsViewModel @Inject constructor(
          */
         private val INLINE_VALIDATION_DELAY = 2.seconds
 
+        fun calculateAgeFromDate(birthDate: DateTime): String {
```

Comment:
i think it would make more sense if this function was moved to the birthdateCalculator class

Reply:
Does this comment mean creating a BirthDateCalculator class.

Reply:
sorry, I meant BirthDatePickerDialog

Reply:
nevermind, leave it as it is